java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture7/tasks/_2/Calculation.java

```diff
@@ -0,0 +1,18 @@
+package com.zoolatech.lecture7.tasks._2;
+
+public abstract class Calculation implements Operations  {
```

Comment:
the class can be removed, cause all children classes have their own logic which completely override all these methods.